lib/web_ui/dev/test_platform.dart

```diff
@@ -130,6 +146,8 @@ class BrowserPlatform extends PlatformPlugin {
   }
 
   Future<shelf.Response> _screeshotHandler(shelf.Request request) async {
+    // Screenshot tests are only enabled in chrome for now.
+    assert(browserName == 'chrome');
```

Comment:
We run the tool itself in release mode, so assertions won't trigger. An alternative that would work is a plain ol' `if () throw`

Reply:
done.